### PR TITLE
write pretrig delta in off files, increment version

### PR DESCRIPTION
**Pull request by ggggggggg**
As discussed I added the pretrig delta to the off files. This should allow the equivalent of the pretrig rms cut in a simple projector independent way.

### CODE REVIEW

File: process_data.go [hunk 1]
```diff
@@ -195,11 +195,18 @@ func (dsp *DataStreamProcessor) AnalyzeData(records []*DataRecord) {
...
+		// slope = dot(x,y.-y[0])/z where z = dot(x,x) and x = [0, 1, 2, ..., N-1]/(N-1), where .-y[0] is elementwise subtraction of the first element
+		np := rec.presamples
+		a := float64(np*(np-1)*(2*np-1)) / float64(6) / float64((np - 1)) // a is the sum of np^2 from 0 to np-1 over (np-1)
+		// a converts valPTDelta to the change over the length of the pretrigger
 		for i := 0; i < rec.presamples; i++ {
 			val += dataVec.AtVec(i)
+			valPTDelta += (dataVec.AtVec(i) - dataVec.AtVec(0)) * float64(i)
 		}
 		ptm := val / float64(rec.presamples)
+		rec.pretrigDelta = valPTDelta / a
```
Comment (joefowler) [inline process_data.go]: I don't think this computation is right. Let me work it out and see if I can show this by further testing.

File: process_data.go [hunk 2]
```diff
@@ -198,15 +198,19 @@ func (dsp *DataStreamProcessor) AnalyzeData(records []*DataRecord) {
...
+		for i := 0; i < npre; i++ {
 			val += dataVec.AtVec(i)
-			valPTDelta += (dataVec.AtVec(i) - dataVec.AtVec(0)) * float64(i)
+			valPTDelta += (dataVec.AtVec(i) - d0) * (float64(i) - xmean)
 		}
-		ptm := val / float64(rec.presamples)
-		rec.pretrigDelta = valPTDelta / a
+		ptm := val / float64(npre)
+		rec.pretrigDelta = valPTDelta * 12.0 / float64(npre*(npre+1))
+		if npre <= 1 {
```
Comment (ggggggggg) [inline process_data.go]: The other assignment should be an else clause here

Comment (joefowler) [reply process_data.go]: Okay. See next commit

**Comment by ggggggggg** [comment]
Thanks. I think merge as soon as tests pass

**Pull request merged**